### ISSUE TITLE
[Bug]: Movie Detail sections reload on scroll

### ISSUE THREAD

**Issue by hadi-norouzi** [moallemi/Film-Time#98]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Related app

All - Android

### What happened?

In MovieDetail page when i scroll to bottom or top of the page sections like Cast and similar reloaded

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by moallemi** [comment]
@hadi-norouzi Yes, I'm aware of that issue. It was done intentionally as part of creating a video to explain how aggressive separation of concerns might end up.
